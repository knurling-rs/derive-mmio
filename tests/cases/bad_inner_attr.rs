#[derive(derive_mmio::Mmio)]
#[repr(C)]
struct Uart {
    #[mmio(RX)]
    status: u32,
}

fn main() {
}
