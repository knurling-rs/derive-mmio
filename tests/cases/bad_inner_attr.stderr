error: `#[mmio(...)]` only supports 'RO', 'RW' and 'inner' options
 --> tests/cases/bad_inner_attr.rs:1:10
  |
1 | #[derive(derive_mmio::Mmio)]
  |          ^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `derive_mmio::Mmio` (in Nightly builds, run with -Z macro-backtrace for more info)
